src/pages/home/report/ReportActionsList.js

```diff
@@ -222,20 +222,28 @@ function ReportActionsList({
     }, [report.lastReadTime, report.reportID]);
 
     useEffect(() => {
-        // If the reportID changes, we reset the userActiveSince to null, we need to do it because
-        // this component doesn't unmount when the reportID changes
-        if (unreadActionSubscription.current) {
-            unreadActionSubscription.current.remove();
-            unreadActionSubscription.current = null;
-        }
-
-        // Listen to specific reportID for unread event and set the marker to new message
-        unreadActionSubscription.current = DeviceEventEmitter.addListener(`unreadAction_${report.reportID}`, (newLastReadTime) => {
+        const resetUnreadMarker = (newLastReadTime) => {
             cacheUnreadMarkers.delete(report.reportID);
             lastReadTimeRef.current = newLastReadTime;
             setCurrentUnreadMarker(null);
+        };
+
+        // Listen to specific reportID for unread event and set the marker to new message
```

Comment:
I just spent a few minutes trying to suggest language improvements in this comment (and the similar one), but this comment just doesn't bring any value.

You can remove this comment and the other one.